src/components/text-editor/link-menu/editor-link-menu.tsx

```diff
@@ -45,14 +53,20 @@ export class TextEditorLinkMenu {
      * (*Not* emitted when the consumer sets the `open`-property to `false`.)
      */
     @Event()
-    private cancel: EventEmitter<void>;
+    private cancel: EventEmitter<string>;
```

Comment:
But this still emits `void` I believe

Reply:
We need this to pass the event.key from the dismiss event. I'm not sure exactly how they are tied in but this error occurs if we don't add the `<string>` to the `EventEmitter`.

The Error:
`Type '(event: CustomEvent<string>) => void' is not assignable to type '(event: LimelTextEditorLinkMenuCustomEvent<void>) => void'.Types of parameters 'event' and 'event' are incompatible.Type 'LimelTextEditorLinkMenuCustomEvent<void>' is not assignable to type 'CustomEvent<string>'.Types of property 'detail' are incompatible.Type 'void' is not assignable to type 'string'.`